p4-16/spec/docs/experimenting-with-p4c-bison-grammar-changes.md

```diff
@@ -0,0 +1,54 @@
+# Introduction
+
+This article describes how to run `bison` on the P4_16 language
+grammar, as implemented in the open source `p4c` compiler available
+at:
+
++ https://github.com/p4lang/p4c
+
+These steps can be useful when trying out variations of the `bison`
+grammar, e.g. when working on making extensions to the P4_16 language
+syntax for enhancements.
+
+Prerequisites: You have a Linux system with the necessary software
+installed required to build `p4c` from source code.
+
+The following sequence of commands can be used to find the particular
+`bison` command line arguments that are used while building `p4c`:
+
+```bash
+git clone https://github.com/p4lang/p4c
+cd p4c
+mkdir build
+cd build
+cmake ..
+find . \! -type d | xargs grep bison
+```
+
+As of 2023-Aug-20, this is the latest version of `p4c`:
```

Comment:
@jafingerhut I'll just merge this, even though I do worry about maintaining this.

Reply:
I mean, it might need updating once every 2 years, and will only be noticed and updated by people who want to use these instructions, which is a pretty small set of people, I expect.